pensieve/analysis.py

```diff
@@ -0,0 +1,100 @@
+import mozanalysis.metrics.desktop as mmd
+from mozanalysis.experiment import Experiment, TimeLimits
+from mozanalysis.bq import BigQueryContext
+from datetime import datetime, timedelta
+import pytz
+import re
+
+
+class Analysis:
+    """
+    Wrapper for analysing experiments.
+    """
+
+    # todo: where should this come from? experiments, metrics?
+    ANALYSIS_PERIOD = 7  # 1 week
+
+    # list of standard metrics to be computed
+    STANDARD_METRICS = [
+        mmd.active_hours,
+        mmd.uri_count,
+        mmd.ad_clicks,
+        mmd.search_count,
+    ]
+
+    def __init__(self, project, dataset):
+        self.project = project
+        self.dataset = dataset
+        self.bq_context = BigQueryContext(project_id=project, dataset_id=dataset)
+
+    def _current_date(self):
```

Comment:
Is this something we'll want to accept through the entrypoint to permit backfills?

Reply:
Eventually, yes.

Reply:
I'll fix it in this PR